lib/rollbar.rb

```diff
@@ -517,7 +517,7 @@ def do_write_payload(payload)
           @file = File.open(configuration.filepath, "a")
         end
 
-        @file.puts payload
+        @file.puts(dump_payload(payload))
```

Comment:
I kinda think that we should do this first, before opening the file, to make things simpler in case the dump fails. Thoughts?